app/controllers/dashboard/projects_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Dashboard
+  # Dashboard Projects Controller
+  class ProjectsController < ApplicationController
+    def index # rubocop:disable Metric/AbcSize
+      @q = Project.ransack(params[:q])
```

Comment:
Within this file and from wherever this is called can you please to update the variable and parameter `q` to something more descriptive

Reply:
`q` is kind of the standard for a query param like this I think

Reply:
`@q` is the standard value that represents the ransacked object. See documentation here https://activerecord-hackery.github.io/ransack/getting-started/simple-mode/

Reply:
Thanks! @joshsadam @ericenns